okpt/io/config/parser.py

```diff
@@ -0,0 +1,31 @@
+"""Provides methods to parse YAML and JSON files."""
+
+import json
+from io import TextIOWrapper
+from typing import Any, Dict
+
+import yaml
+
+from okpt.io import reader
+
+
+def parse_yaml(file: TextIOWrapper) -> Dict[str, Any]:
+    """Parses YAML file from file object."""
+    return yaml.load(file, Loader=yaml.SafeLoader)
+
+
+def parse_yaml_from_path(path: str):
```

Comment:
Why doesn't this function have a type hint for return?

Reply:
Since `parse_yaml_from_path` uses `parse_yaml` (which has a type hint for return), the return can be inferred by most type checkers.

Reply:
I think for consistency, you should add them.

Reply:
Done, I've updated some others as well.